server/index.js

```diff
@@ -3,28 +3,70 @@ const bodyParser = require('body-parser');
 const path = require('path');
 
 const app = express();
-const db = require('../database/controllers');
-
-console.log(db);
+const {
+  getDoc, saveDoc, updateDoc, deleteDoc,
+} = require('../database/controllers');
 
 const PORT = 3002;
 
 app.use(bodyParser.json());
 app.use(bodyParser.urlencoded({ extended: true }));
 app.use(express.static(path.join(__dirname, '/../public')));
 
+app.post('/items/', (req, res) => {
+  saveDoc(req.body, (err) => {
+    if (err) {
+      console.log(`error saving: ${err}`);
+      res.status(400);
```

Comment:
If you want to condense, another option is to do res.sendStatus(400) and then you don't need the res.end()